### ISSUE TITLE
When a Dialog or FormApplication instance is rendered which contains an input that declares the `autofocus` property, automatically focus that field after rendering.

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#6819]
A small accessibility gain which also improves workflow for bulk creation. 

Currently, whenever a new window spawns in FVTT (such as many of the document creation workflows) the newly created window does not receive Focus automatically, which means if you are fine with the defaults you still have to click into the window to press enter and accept them.

By triggering a focus event on newly rendered Dialog and FormApplication class elements, there's a marked increase in speed of data entry.
